blocks/pxt_blockly_functions.js

```diff
@@ -463,6 +463,11 @@ Blockly.PXTBlockly.FunctionUtils.getShadowBlockInfoFromType_ = function(argument
       fieldName = 'TEXT';
       fieldValue = 'abc';
       break;
+    case 'Array':
+      shadowType = 'variables_get';
+      fieldName = "VAR";
+      fieldValue = Blockly.Variables.getOrCreateVariablePackage(ws, null, "list", '').getId();
```

Comment:
is there a reason for this to be it's own case instead of an entry in the argumentDefaultNames map?

Reply:
(small) and should it use `Blockly.Msg.FUNCTIONS_DEFAULT_ARRAY_ARG_NAME` ?

Reply:
oh I just thought since it was going to be one of our default types instead of a custom type, it'd be nice to break it out, but I can be convinced either way!